authors/apps/articles/urls.py

```diff
@@ -12,5 +12,7 @@
     path('articles/<slug>/rate/', views.RatingDetails.as_view(), name='ratings'),
     path('articles/<slug>/favourite', views.FavouriteGenericAPIView.as_view(), name="favourite"),
     path('tags/', views.TagsView.as_view(), name='tags'),
-    path('articles/<slug>/like/', views.ArticlesLikesDislikes.as_view(),  name='article-like')
+    path('articles/<slug>/like/', views.ArticlesLikesDislikes.as_view(),  name='article-like'),
+    path('articles/statistics/', views.ArticleStatsView.as_view(), name='stats'),
```

Comment:
Hey @gloriaodipo, kindly drop one of these paths, it appears to be repeated. Thank you.

Reply:
Okay, thanks for the correction